Makefile

```diff
@@ -1,6 +1,7 @@
 default: base
 
 NODE_CONFIG_ENV ?= test
+export PGAPPNAME ?= odkcentral
```

Comment:
It loo looks like it will set the `application_name` for individual connections created from processes started from `make`.

I don't think this will have any effect in production.